src/extensions/amazon-sales/index.md

```diff
@@ -3,11 +3,14 @@ group: extensions
 title: Install Amazon Sales Channel
 ---
 
+{:.bs-callout .bs-callout-warning}
+Amazon Sales Channel versions 4.0+ are only supported when integrated with Magento 2.3.x.
+
 The Amazon Sales Channel extension installs and adds features to integrate your Magento catalog with Amazon Seller Accounts to sell through the Amazon Marketplace. To review additional information, see the [Amazon Sales Channel Marketplace](http://marketplace.magento.com/magento-module-amazon.html) page.
 
 ## Requirements
 
--  **Magento Instance**: Amazon Sales Channel can be installed on instances with {{site.data.var.ce}} and {{site.data.var.ee}} versions 2.2.4+ and 2.3.X. We do not support the extension on Magento 2.1 or Magento 1.
+-  **Magento Instance**: Amazon Sales Channel can be installed on instances with {{site.data.var.ce}} and {{site.data.var.ee}} versions 2.3.x. We no longer support the extension on Magento 2.1, Magento 2.2, or Magento 1.
```

Comment:
The release notes indicate support for ece as well. Should that be added here?

Reply:
updated to match both places

Reply:
I don't see any change.

Reply:
i haven't pushed the commit yet